src/More/zendeskAPI.ts

```diff
@@ -0,0 +1,75 @@
+import env from "react-native-config"
+
+const defaultHeaders = {
+  "content-type": "application/json",
+}
+
+type EnvironmentProps = {
+  os: string
+  osVersion: string
+  appVersion: string
+}
+
+type FeedbackProps = {
+  subject: string
+  name: string
+  body: string
+  environment: EnvironmentProps
+}
+
+interface NetworkSuccess {
+  kind: "success"
+}
+
+interface NetworkFailure<U> {
+  kind: "failure"
+  error: U
+}
+
+export type FeedbackError = "Unknown" | "ZendeskError"
+
+export type NetworkResponse<U = "Unknown"> = NetworkSuccess | NetworkFailure<U>
+
+const OS_FIELD_KEY = "360033622032"
```

Comment:
Are these zendesk specific things? will they ever change in the future? should they be included in the .env?

Reply:
I don't know if they will change in the future, but if they do we will most likely have to make changes to the implementation. Because if the concept of the field changes I would think that the value should be different also.